### PR TITLE
Adding additional user info to user page

### PR DESCRIPTION
**Pull request by ethan-nelson**
#326 lists a few things that would be great to have on the user page regarding a user's OSM history. Also #300 alludes to some of this. I've added to the user profile:

* User join date (hovering over gives full string)
* Total changesets
* Direct link to OSM edits

users

When you are viewing a user page that's not your own, the join date string should be in line with the profile link.

### CODE REVIEW

**Comment by ethan-nelson** [comment]
Could probably use some better error handling in the event it cannot get a response from the OSM API, but took a first jab at it.

**Comment by pyrog** [comment]
> Total changesets

It could interesting to display also the numbers of changesets made for hot.
Search changesets with comments start with **#hotosm-task**

**Comment by bgirardot** [comment]
Just to note, recent TM2 updates changed #hotosm-task to #hotosm-project for newly created projects to match the naming convention we use, program -> project -> task

**Comment by ethan-nelson** [comment]
@pyrog yes it would be great to have HOT changesets, but I feel it's hard to discern fully those that count for that. It wouldn't count any changesets someone made without those labels (I know I did that many times with v1 tasks) or ones with the url enconding issue. And also I don't know if it's a fully representative metric. Sometimes people upload a single changeset for every node edit while others do only one for a full task. That said, I don't know of a fairer or more accurate metric either...perhaps number of objects edited within the bboxes of every task they worked on or total up the number of tasks over all the projects. I'd make a new issue for this request specifically as it will require much more code anyway.

File: osmtm/templates/user.mako [hunk 1]
```diff
@@ -29,6 +29,30 @@ else:
           <img src="http://www.openstreetmap.org/favicon.ico" alt="[OSM]" />${_('OSM Profile')}</a>
       </p>
     </div>
+    <div class="col-md-6">
+    % if creation_date != 'null':
+    <p title=${creation_date}>
+       ${_('This user joined OSM on ${join_date}.', mapping={'join_date':creation_date[5:7] + \
+                                                               '/' + creation_date[8:10] + '/' + \
+                                                               creation_date[0:4]})}
+    </p>
```
Comment (pgiraud) [inline osmtm/templates/user.mako]: This would be great to have localized dates.

Comment (ethan-nelson) [inline osmtm/templates/user.mako]: Acknowledged. Will work on this.

Comment (ethan-nelson) [inline osmtm/templates/user.mako]: Haven't forgotten about this :) got busy with work, but will get back to this soon.

**Comment by ethan-nelson** [comment]
Closing to reopen for i18n addition.

**Pull request closed without merging**